wodles/aws/buckets_s3/waf.py

```diff
@@ -20,6 +31,23 @@ class AWSWAFBucket(AWSCustomBucket):
     def __init__(self, **kwargs):
         kwargs['db_table_name'] = 'waf'
         AWSCustomBucket.__init__(self, **kwargs)
+        if self.check_waf_type():
+            self.service = 'WAFLogs'
+            self.type = WAF_NATIVE
+            self.empty_bucket_message_template = AWSBucket.empty_bucket_message_template
+        else:
+            self.type = WAF_KINESIS
+
+    def check_waf_type(self):
+        try:
```

Comment:
Add a docstring here to describe this method since it could be a bit confusing. What does it mean when it returns True? Is it native or using Kinesis? and what when it returns False? It can be inferred, but it is better to explain it.